### ISSUE TITLE
importing extensions broken in dev

### ISSUE THREAD

**Issue by bendichter** [hdmf-dev/hdmf#580]
## Description

https://github.com/hdmf-dev/hdmf/pull/555 caused import to break on many extensions.

## Steps to Reproduce

```python

import ndx_grayscalevolume

---------------------------------------------------------------------------
KeyError                                  Traceback (most recent call last)
<ipython-input-3-f30be7452514> in <module>
----> 1 import ndx_grayscalevolume

~/opt/anaconda3/lib/python3.7/site-packages/ndx_grayscalevolume/__init__.py in <module>
     10 load_namespaces(ns_path)
     11 
---> 12 GrayscaleVolume = get_class('GrayscaleVolume', name)

~/dev/hdmf/src/hdmf/utils.py in func_call(*args, **kwargs)
    582             def func_call(*args, **kwargs):
    583                 pargs = _check_args(args, kwargs)
--> 584                 return func(**pargs)
    585 
    586         _rtype = rtype

~/opt/anaconda3/lib/python3.7/site-packages/pynwb/__init__.py in get_class(**kwargs)
    185     """
    186     neurodata_type, namespace = getargs('neurodata_type', 'namespace', kwargs)
--> 187     return __TYPE_MAP.get_container_cls(namespace, neurodata_type)
    188 
    189 

~/dev/hdmf/src/hdmf/utils.py in func_call(*args, **kwargs)
    578             def func_call(*args, **kwargs):
    579                 pargs = _check_args(args, kwargs)
--> 580                 return func(args[0], **pargs)
    581         else:
    582             def func_call(*args, **kwargs):

~/dev/hdmf/src/hdmf/build/manager.py in get_container_cls(self, **kwargs)
    503         cls = self.__get_container_cls(namespace, data_type)
    504         if cls is None and autogen:  # dynamically generate a class
--> 505             spec = self.__ns_catalog.get_spec(namespace, data_type)
    506             self.__check_dependent_types(spec, namespace)
    507             parent_cls = self.__get_parent_cls(namespace, data_type, spec)

~/dev/hdmf/src/hdmf/utils.py in func_call(*args, **kwargs)
    578             def func_call(*args, **kwargs):
    579                 pargs = _check_args(args, kwargs)
--> 580                 return func(args[0], **pargs)
    581         else:
    582             def func_call(*args, **kwargs):

~/dev/hdmf/src/hdmf/spec/namespace.py in get_spec(self, **kwargs)
    314         namespace, data_type = getargs('namespace', 'data_type', kwargs)
    315         if namespace not in self.__namespaces:
--> 316             raise KeyError("'%s' not a namespace" % namespace)
    317         return self.__namespaces[namespace].get_spec(data_type)
    318 

KeyError: "'GrayscaleVolume' not a namespace"
```

I get a similar error with all extensions I have tried except ndx_events

## Environment

hdmf from https://github.com/hdmf-dev/hdmf/pull/555 to latest

**Comment by t-b** [comment]
Once that is fixed. How about testing hdmf dev with all extensions from the catalogue in CI?

**Comment by rly** [comment]
This is fixed in 2.5.1 and in the dev branch. @t-b, I added #592 on your suggestion.
